saleor/attribute/utils.py

```diff
@@ -51,6 +51,13 @@ def associate_attribute_values_to_instance(
             assignment_id=assignment.pk
         ).update(product_id=instance.pk)
 
+    # This code will be deleted in new release, it is temporary solution between
+    # releases to keep database in sync
```

Comment:
IMO we should write the specific version number here

Reply:
done